recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-dom0.dts

```diff
@@ -69,6 +69,46 @@
 		      <0x6 0x00000000 0x0 0x80000000>;
 	};
 
+	mailbox: mailbox@0 {
+		compatible = "arm,smc-mbox";
+		#mbox-cells = <1>;
+		arm,func-ids = <0x82000001>, <0x82000002>;
+		method = "smc";
+	};
+
+	sram: sram@47FFF000 {
+		compatible = "mmio-sram";
+		/*compatible = "renesas,rcar-sram-ns", "mmio-sram";*/
```

Comment:
Why do you need commented code in the repo?

Reply:
I can drop these comments. We have shared page being located in DDR (because mmio-sram is entirely used by secure world). The only only reason I decided to leave them is to remind that protocol mentions it should be in mmio-sram.

Reply:
If you want to remind something - then add corresponding comment. You don't need to leave pieces of commented out code.

Reply:
will drop

Reply:
just put a comment

Reply:
This comment was removed.